internal/apiserver/apiserver.go

```diff
@@ -33,29 +35,47 @@ func Run() {
 	if err != nil {
 		log.Fatal(err)
 	}
-
+	
 	idGen, err := idgen.New(cfg.SnowflakeNode)
 	if err != nil {
 		log.Fatal(err)
 	}
-
-	repos := fake_repo.New()
-
+	
+	pgConfig := postgres.NewPgConfig(
+		cfg.Postgres.User,
+		cfg.Postgres.Password,
+		cfg.Postgres.Host,
+		cfg.Postgres.Port,
+		cfg.Postgres.Database,
+	)
+	
+	pgClient, err := postgres.NewClient(pgConfig)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Shouldn't we use defer to close the client here? We don't have a graceful shutdown yet, but we will eventually